draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -276,55 +274,57 @@
-				<t>Type: 1.</t>
-				<t>Length: 4 octets.</t>
-				<t>Value: number of un-acknowledged LSPs which can be sent back to back.</t>
-				<t>Note that if an LSP has not been acknowledged and is sent again, it does not count twice. The reason is that this LSP is assumed to be lost and hence not in a buffer at the receiver.</t>
+			<t>For a given IS-IS adjacency, the Flooding Parameters TLV does not need to be advertised in each IIH or PSNP. The LSP transmitter uses the latest received value for each parameter until a new value is advertised by the LSP receiver. However, IIHs and PSNPs are not reliably exchanged, and may never be received, so parameters MAY be sent even if there is no change in value since the last transmission. For a parameter which has never been advertised, the LSP transmitter SHOULD use its local default value. That value SHOULD be configurable on a per node basis and MAY be configurable on a per interface basis.</t>
+			<section anchor="LSPReceptionWindow" title="LSP Burst Window sub-TLV">
+				<t>The LSP Burst Window sub-TLV advertises the maximum number of un-acknowledged LSPs that the node can receive with no separation interval between LSPs.</t>
+				<t>Type: 1</t>
+				<t>Length: 4 octets</t>
+				<t>Value: number of un-acknowledged LSPs that can be sent back to back</t>
+				<t>Note that if an LSP has not been acknowledged and is sent again, it does not count twice. The reason is that this LSP is assumed to be lost and hence not in a buffer at the receiver.
+				[This requires some clarification. Why would we send the same LSP twice in the same burst? If it is a retransmission of an LSP that was originally sent outside of the current burst, then shouldn't it count as part of the burst? -- Tony L.]</t>
+
 			</section>
-			<section anchor="InterfaceLSPTransmissionInterval" title="minimumInterfaceLSPTransmissionInterval sub-TLV">
-				<t>The sub-TLV minimumInterfaceLSPTransmissionInterval advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be processed/received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
-				<t>Type: 2.</t>
-				<t>Length: 4 octets.</t>
-				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the receive window has been used.</t>
+			<section anchor="InterfaceLSPTransmissionInterval" title="LSP Transmission Interval sub-TLV">
+				<t>The LSP Transmission Interval sub-TLV advertises the minimum interval, in micro-seconds, between LSPs arrivals which can be received on this interface, after the maximum number of un-acknowledged LSPs has been sent.</t>
+				<t>Type: 2</t>
+				<t>Length: 4 octets</t>
+				<t>Value: minimum interval, in micro-seconds, between two consecutive LSPs sent after the burst window has been used</t>
+				<t>The LSP Transmission Interval is an advertisement of the receiver's steady-state LSP reception rate.</t>
```

Comment:
NOTE: The total number of LSPs sent/second (including bursts) SHOULD NOT exceed 1000000/minimum interval.

Reply:
@ Les: no.cf above.
@ Tony: I'd say receiver's minimal steady-state.  (I would assume this number be 'large' e.g. 33 ms as per today even though probably many implementations having a small number of IS-IS adjacencies would probably be capable of handling 10ms.

Reply:
@Les: Sorry, I don't understand your comment.  Why do we need an additional restriction?
@Bruno: Saying 'minimal' then begs the question of what is 'maximal'.  We are not advertising a range.

Reply:
I don't see why this calls for a 'maximal' (e.g. you can enter the charity concert with a minimum donation 20$; does not seem to require the specification of a max) .
Also the semantic is already defined in the ISO spec as "Minimum interval between transmission of LSPs on a broadcast circuit (See clause 7.3.15.6)."